tox.ini

```diff
@@ -52,20 +52,30 @@ basepython =
     py310: python3.10
     py311: python3.11
 
+[blocklint]
+max_issue_threshold=1
+blocklist=blacklist,whitelist,slave
+exactlist=master ; allow URLs to contain some terms
+
 [common-lint]
 extras=linter
+allowlist_externals=
+    /bin/bash
```

Comment:
I don't see that we're using bash anywhere within the lint tasks so I think this can go.

Reply:
Ended up needing bash 😭 

It will be scorched from existence with pre-commit.